app/services/webhook_manager/gift_card_creator.rb

```diff
@@ -29,7 +29,11 @@ def call
           recipient: payment_intent.recipient,
           single_use: single_use
         )
-        GiftCardAmount.create!(value: amount, gift_card_detail: gift_card_detail)
+        amount_after_fees = WebhookManager::FeeHandler.call({
```

Comment:
This feels like a weird way to call this utility--instantiating the class then using it's member variables for processing, can we not just define a method that accepts these as arguments?